terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -845,6 +855,33 @@ then
       validation_return_code=2
     fi
   fi
+
+  if [ "$secure_mode" = "true" ]; then
+    res=$(validate_subnet_port_access ${WLS_SUBNET_OCID} ${MS_ADMIN_PORT} ${wls_subnet_cidr_block})
+    if [[ $res == *"WARNING"* ]]
+    then
+      for warning in "${res[@]}"; do
+        echo "$warning"
+      done
+    elif [[ $res -ne 0 ]]
+    then
+      echo "ERROR: Port ${MS_ADMIN_PORT} is not open for access by WLS Subnet CIDR [$wls_subnet_cidr_block] in WLS Subnet [$WLS_SUBNET_OCID]. ${NETWORK_VALIDATION_MSG}"
+      validation_return_code=2
+    fi
+  fi
+   if [ "$secure_mode" = "true" ]; then
+    res=$(validate_subnet_port_access ${WLS_SUBNET_OCID} ${WLS_NM_PORT} ${wls_subnet_cidr_block})
```

Comment:
Why is a check for node manager port access only for secured production mode? Wouldn't this be needed for all WebLogic installations?